Doppler.HtmlEditorApi.Test/SaveCampaignTest.cs

```diff
@@ -463,6 +473,16 @@ public async Task PUT_campaign_should_store_unlayer_content(int? currentEditorTy
                         && c.Meta == metaAsString)))
                 .ReturnsAsync(1);
 
+            dbContextMock
+                .Setup(x => x.ExecuteAsync(
+                    It.Is<string>(s => s.Trim().StartsWith("UPDATE Campaign")),
+                    It.Is<UpdateCampaignStatusDbQuery.Parameters>(c =>
+                        c.setCurrentStep == 2
+                        && c.setHtmlSourceType == 2
```

Comment:
I am not sure if using `2` is fine because it means `Template`.

https://github.com/MakingSense/Doppler/blob/48cf637bb1f8b4d81837fff904d8736fe889ff1c/Doppler.Transversal/Classes/CampaignHTMLContentTypeEnum.cs#L12-L17

At the moment we do not have templates, but we will have them soon. So, I think that `2` is enough good, but I will add a comment in the code.

Reply:
Why don't use 3 (Editor)? And use EditorType on Content

Reply:
We talked with @MarianoHerrera, it is not a priority, probably 3 is better I will try. Anyway, I will add a comment because it is going to be changed in the near future.

Reply:
Done! 

I used 3 and added a reference to the Doppler source code.

Marking this comment as solved...

Reply:
@MarianoHerrera, finally, after test it I decided to use `2 (Template)` in place of `3 (Editor)` because Doppler seems to understand `3 (Editor)` as related to _the old HTML_ editor.